Setup/Patch/Data/AddLifeElementsValidationValuePatch.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bold\Checkout\Setup\Patch\Data;
+
+use Magento\Framework\Serialize\Serializer\Json;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+/**
+ * Perform Life elements upgrade.
+ */
+class AddLifeElementsValidationValuePatch implements DataPatchInterface
+{
+    private const PATH_LIFE_ELEMENTS = 'checkout/bold_checkout_life_elements/life_elements';
+
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+
+    /**
+     * @var Json
+     */
+    private $serializer;
+
+    /**
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     * @param Json $serializer
+     */
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup,
+        Json                     $serializer
+    ) {
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->serializer = $serializer;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function getDependencies()
+    {
+        return [];
+    }
+
+    /**
+     * Perform Life elements upgrade.
+     *
+     * @return void
+     */
+    public function apply(): void
+    {
+        $this->moduleDataSetup->startSetup();
+
+        $allLifeElements = $this->getLifeElements();
+        $result = $this->updateLifeElements($allLifeElements);
+        $this->setLifeElements($result);
+
+        $this->moduleDataSetup->endSetup();
+    }
+
+    /**
+     * Get existing Life elements from database.
+     *
+     * @return array
+     */
+    public function getLifeElements(): array
+    {
+        $connection = $this->moduleDataSetup->getConnection();
+        $table = $connection->getTableName('core_config_data');
+        $select = $connection->select()->from(
+            $table,
+            [
+                'scope',
+                'scope_id',
+                'path',
+                'value',
+            ]
+        )->where(
+            'path = ?',
+            self::PATH_LIFE_ELEMENTS
+        );
+
+        return $connection->fetchAll($select);
+    }
+
+    /**
+     * Save updated Life elements to database.
+     *
+     * @param array $result
+     * @return void
+     */
+    public function setLifeElements(array $result): void
+    {
+        $connection = $this->moduleDataSetup->getConnection();
+        $table = $connection->getTableName('core_config_data');
+        $connection->insertOnDuplicate(
+            $table,
+            $result
+        );
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getAliases()
+    {
+        return [];
+    }
+
+    /**
+     * Add to each element a 'input_regex' if it is absent.
+     *
+     * @param array $allLifeElements
+     * @return array
+     */
+    public function updateLifeElements(array $allLifeElements): array
```

Comment:
Maybe we need to create a service to do this job for future updates and just use it for additional fields to add?

Reply:
I don't think this should be done in the scope of this task.